<?xml version="1.0" encoding="utf-8"?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>MariaDB Knowledge Base Comments for: Character conversion fails getting columns or restult sets</title><link>https://mariadb.com/kb/en/character-conversion-fails-getting-columns-or-restult-sets/+comments/feed/</link><description></description><atom:link href="https://mariadb.com/kb/en/character-conversion-fails-getting-columns-or-restult-sets/+comments/feed/" rel="self"></atom:link><language>en-us</language><lastBuildDate>Mon, 13 Jan 2025 06:37:16 +0000</lastBuildDate><item><title>Re: Character conversion fails getting columns or restult sets</title><link>https://mariadb.com/kb/en/character-conversion-fails-getting-columns-or-restult-sets/+comments/5425</link><description>&lt;p&gt;Package: odbc-mariadb
Version: 3.1.9-1build1&lt;/p&gt;
&lt;p&gt;Just installed this and I have the same problem on PopOS (Ubuntu)&lt;/p&gt;
&lt;p&gt;cursor.tables()
pyodbc.Error: ('HY000', '[HY000] [ma-3.1.9][5.6.38]Ansi to Unicode conversion error occurred (95) (SQLDescribeCol)')&lt;/p&gt;
</description><dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">Jack J. Woehr</dc:creator><guid>https://mariadb.com/kb/en/character-conversion-fails-getting-columns-or-restult-sets/+comments/5425</guid></item></channel></rss>